modules/ui/background.js

```diff
@@ -283,6 +288,77 @@ export function uiBackground(context) {
             .text(t('background.imagery_source_faq'));
     }
 
+    function renderLayerToggle() {
+        const getlist = Object.keys(Hoot.layers.loadedLayers).map( d => Hoot.layers.loadedLayers[d] );
+
+        _layerToggleList.html('');
+
+        let updateList = _layerToggleList.selectAll('.layer-toggle-item')
+            .data(getlist);
+
+        updateList.exit().remove();
+
+        updateList = updateList.enter()
+            .append('li')
+            .attr('class', 'layer layer-toggle-item')
+            .merge(updateList);
+
+        let updateButton = updateList.selectAll('.lyrcntrlbtn')
+            .data(d => [d]);
+
+        updateButton.exit().remove();
+
+        updateButton = updateButton.enter()
+            .append('button')
+            .call(tooltip().title('Click to have layer be on top').placement('left'))
```

Comment:
sick! Though I think we don't want to have this for the top most layer. I think we can use the `.each()` here method which gives us the index so we can ignore the 0th....

I think it's something like

```javascript
.each(function(d, i) {
     if (i===0) return
     d3.select(this).call(/*the tooltip stuff you have*/)
}